include/finufft/defs.h

```diff
@@ -65,86 +64,96 @@
 // ------------- Library-wide algorithm parameter settings ----------------
 
 // Library version (is a string)
-#define FINUFFT_VER          "2.2.0"
+#define FINUFFT_VER "2.2.0"
 
 // Smallest possible kernel spread width per dimension, in fine grid points
 // (used only in spreadinterp.cpp)
-#define MIN_NSPREAD          2
+static constexpr int MIN_NSPREAD = 2;
```

Comment:
I do not think variables in a header should be static as a new instance of the variable is created every time the header is included. Am I missing something here?

Reply:
I think this is not an issue with anything declared `constexpr`, but I'll do more reading. In any case, this should be fixable by adding `inline`.

Reply:
None of these variables should ever materialize in the code, they will be directly replaced by their value, analogously to the original macros.

Reply:
You are right. The consensus seems to be that the proper definition for this kind of constant should be `inline constexpr` (the `static` is already implied). I'll fix this.

Reply:
I'll have a look

Reply:
I think I fixed it :)